src/main/java/com/expedia/alertmanager/notifier/AwsSesNotifier.java

```diff
@@ -39,62 +25,31 @@ public AwsSesNotifier(String to, String from) {
 
     @Override
     public void execute(MappedMetricData mappedMetricData) {
-        final String defaultCharSet = MimeUtility.getDefaultJavaCharset();
         try {
-            final Session session = Session.getDefaultInstance(new Properties());
             final String subject = "Alert for " + mappedMetricData.getMetricData().getMetricDefinition().getKey();
-            final MimeMessage message = new MimeMessage(session);
-            message.setSubject(subject, "UTF-8");
-            message.setFrom(new InternetAddress(to));
-            message.setRecipients(Message.RecipientType.TO, InternetAddress.parse(from));
-            message.setContent(buildMimeMultipart(mappedMetricData, defaultCharSet));
-
-            final AmazonSimpleEmailService client = AmazonSimpleEmailServiceClientBuilder.standard()
-                                                        .withRegion(Regions.US_WEST_2).build();
-
-            final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-            message.writeTo(outputStream);
-            final RawMessage rawMessage =
-                new RawMessage(ByteBuffer.wrap(outputStream.toByteArray()));
-
-            final SendRawEmailRequest rawEmailRequest =
-                new SendRawEmailRequest(rawMessage);
-
-            final SendRawEmailResult rawEmailResult = client.sendRawEmail(rawEmailRequest);
-            log.info("Email sent status: " + rawEmailResult);
+            final String body = "Anomaly Result " + mappedMetricData.getAnomalyResult().toString();
+            AmazonSimpleEmailService client =
+                AmazonSimpleEmailServiceClientBuilder.standard()
+                    .withRegion(Regions.US_WEST_2).build();
+            SendEmailRequest request = new SendEmailRequest()
+                .withDestination(
+                    new Destination().withToAddresses(to))
+                .withMessage(new Message()
+                    .withBody(new Body()
+                        .withHtml(new Content()
+                            .withCharset("UTF-8").withData(body))
+                        .withText(new Content()
+                            .withCharset("UTF-8").withData(body)))
+                    .withSubject(new Content()
+                        .withCharset("UTF-8").withData(subject)))
+                .withSource(from);
+            final SendEmailResult sendEmailResult = client.sendEmail(request);
+            log.info("Email sent status: " + sendEmailResult);
```

Comment:
Use string interpolation for slf4j log messages (avoids unnecessary concatenation when the log level is a no-match):

```
log.info("Email sent status: {}", sendEmailResult);
```